components_library/boards/test_board.py

```diff
@@ -61,12 +68,12 @@ def __init__(
             clock=clk_freq, voltage_domain=VoltageDomain()
         )
 
-        self.mem_ranges = memory.get_memory_ranges()
-
     def connect_system_port(self, port: Port) -> None:
         self.system_port = port
 
     def connect_things(self) -> None:
```

Comment:
Is there a good reason to not move this implementation to the `AbstractBoard`? This seems like pretty default behavior (I know X86 will need overridden regardless).

Reply:
Yeah, you're right. That was a step I forgot to take.